about.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style/style1.css">
+    <title>About</title>
+</head>
+
+<body>
+    <header class="top">
+        <ul>
+            <li><a><i class="fa-brands fa-facebook"></i></a></li>
+            <li><a><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a>Logout</a></li>
+            <li><a>Mypage</a></li>
+            <li><a href="https://en.wikipedia.org/wiki/Morocco">morocco</a></li>
+        </ul>
+    </header>
+
+<!-- mobile menu -->
+<div class="menuformobile">
+    <img class="closeicon"  onclick="CloseMenu()" src="style/images/Icon.png" alt="close_icon">
+    <div class="top-menu">
+        <ul>
+            <li><a><i class="fa-brands fa-facebook"></i></a></li>
+            <li><a><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a>Logout</a></li>
+            <li><a>Mypage</a></li>
+            <li><a href="https://en.wikipedia.org/wiki/Morocco">Morocco</a></li>
+        </ul>
+    </div>
+    <ul class="mobile-menu">
+        <li><a href="./index.html" class="links" id="homeLink" >Home</a></li>
+        <li><a href="./about.html" class="links" id="aboutLink" >About</a></li>
+        <li><a id="programLink" class="links">Program</a></li>
+        <li><a id="joinLink" class="links">Join</a></li>
+        <li><a id="contactLink" class="links">Contact</a></li>
+    </ul>
+</div>
+<!-- End -->
+
+<main>
+    <section class="headline">
+        <nav class="navbar">
+            <i class="fa-solid fa-bars"  onclick="OpenMenu()" id="menu-icon"></i>
+            <div class="desktop-menu">
+                   <a id="logo" id="homeLink" href="./index.html"> <img src="style/images/logo.png" alt="logo"></a>
+                <ul>
+                     <li><a href="./about.html" id="aboutLink" id="active">About</a></li>
+                    <li><a href="#">Program</a> </li>
+                    <li><a href="#">Join</a> </li>
+                    <li><a href="#">Contact</a> </li>
+                    <li><a href="#" class="compaign">Compaign</a> </li>
+                </ul>
+            </div>
+        </nav>
+    </section>
```

Comment:
- [ ] The navbar is part of the header and would be better fit inside of the `header` tag with the top dark bar. Kindly change the position of your navbar, it also should not be wrapped in a `section` tag. The navbar consists of a top dark bar and a bottom white bar.
 - [ ] Kindly fix the sizing of your navbar. There are items on both the dark bar and white bar that are being cut out of the screen on sizes close to the transition from desktop to mobile view.
 - Your design:
 - 
<img width="862" alt="image" src="https://user-images.githubusercontent.com/61919617/203772514-44ac6a53-ad55-4df3-a688-bcc9f26d1fbc.png">